versioned/persist/tx/src/main/java/org/projectnessie/versioned/persist/tx/TxDatabaseAdapter.java

```diff
@@ -178,6 +179,7 @@ public Stream<CommitLogEntry> commitLog(Hash offset) throws ReferenceNotFoundExc
     ConnectionWrapper conn = borrowConnection();
     boolean failed = true;
     try {
+      @SuppressWarnings({"StreamResourceLeak", "MustBeClosedChecker"})
```

Comment:
shouldnt this method also be annotated with `@MustBeClosed` ?
we are even returning `return intLog.onClose(conn::close);` so this method itself is adding on more "cleanup-responsibility" to the inner stream

same comment for all other suppression added to this file